text/0000-result-expect.md

```diff
@@ -0,0 +1,40 @@
+- Feature Name: `result_expect`
+- Start Date: 2015-05-13
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Add an `expect` method to the Result type, bounded to `E: Debug`
```

Comment:
The bound should probably be `Display`.
`unwrap` and `Debug` aren't meant to reach the user, in that context adding a function that shows a message is pointless. If it's `Display`, this would remove the need to do `panic!("error: {}", e);` ([example](https://github.com/rust-gnome/glib/blob/master/glib-sys/build.rs#L4-7)).